### PR TITLE
feat: added Modal component

### PR DESCRIPTION
**Pull request by kedark152**
### #feat: added Modal Component.
1. Basic Modal structure.
2. Demo Modal display on button click.

**Preview Link:** https://deploy-preview-6--forest-ui.netlify.app/docs-site/docs

### CODE REVIEW

**Review by Raghav888 — APPROVED** [review]

File: docs-site/docs.css [hunk 1]
```diff
@@ -16,6 +16,7 @@
 @import url(../stylesheets/components/input.css);
 @import url(../stylesheets/components/text-utils.css);
 @import url(../stylesheets/components/lists.css);
+@import url(../stylesheets//components/modal.css);
```
Comment (fahd23) [inline docs-site/docs.css]: Is this working? you have // in it

**Review by fahd23 — COMMENTED** [review]
Code looks good.

File: docs-site/docs.css (hunk 1 above)
Comment (kedark152) [reply docs-site/docs.css]: Yes it is working, check the preview @ https://deploy-preview-6--forest-ui.netlify.app/docs-site/docs.
Still I have removed it.

Comment (fahd23) [reply docs-site/docs.css]: Ok then, good to merge

**Review by fahd23 — APPROVED** [review]

**Pull request merged**